crates/cheatnet/tests/cheatcodes/declare.rs

```diff
@@ -1,8 +1,15 @@
 use crate::common::state::create_cheatnet_state;
 use crate::common::{get_contracts, state::create_cached_state};
-use cairo_felt::Felt252;
 use cheatnet::cheatcodes::{CheatcodeError, EnhancedHintError};
 use conversions::StarknetConversions;
+use starknet_api::core::ClassHash;
+use starknet_api::hash::StarkFelt;
+use starknet_api::stark_felt;
+
+static HELLO_STARKNET_CLASS_HASH: &str =
+    "0x298f80e468953d1e65314b6bc63347c7a3fe454a89c2b15387dd52ee281d103";
```

Comment:
Oh I think I misguided you, I thought you mean hardcoding declared contracts in cairo code as those stay on chain forever. In such case as here it will change with each sierra version bump.

Reply:
I think it is still good to have such test, though it will be pain to re-calculate it each time with sncast. Maybe we could use `starknet.rs` to automate it here?

Reply:
It is ok to merge this one and add another issue for it

Reply:
added https://github.com/foundry-rs/starknet-foundry/issues/929